cli/command/formatter/volume.go

```diff
@@ -45,40 +45,59 @@ func VolumeWrite(ctx Context, volumes []*types.Volume) error {
 		}
 		return nil
 	}
-	return ctx.Write(&volumeContext{}, render)
+	return ctx.Write(newVolumeContext(), render)
+}
+
+type volumeHeaderContext map[string]string
+
+func (c volumeHeaderContext) Label(name string) string {
```

Comment:
Thinking outloud 🤔 but I think these are duplicated a lot, we might be able to do some "refactoring" 👼